Documentation/Commands/colors.md

```diff
@@ -0,0 +1,24 @@
+# Colors parser
+
+## Input
+
+The colors parser supports multiple input file types:
+
+ - CLR: NSColor​List palette file ([see docs](https://developer.apple.com/reference/appkit/nscolorlist)). When you create color palettes in image editors or in Xcode, these are stored in `~/Library/Colors`.
+ - JSON: Simple root object where each key is the name, each value is a hex color string.
+ - TXT: Each line has a name and a color, separated by a `:`. A color can either be a color hex value, or the name of another color in the file.
+ - XML: Android colors.xml file parser ([see docs](https://developer.android.com/guide/topics/resources/more-resources.html#Color)).
+
+### Filter
+
+The default filter for this command is: `[^/]\.(?i:clr|json|txt|xml)$`. That means it'll accept any file with the extensions `clr`, `json`, `txt` or `xml`.
+
+You can provide a custom filter using the `filter` option, it accepts any valid regular expression. See the [Config file documentation](../ConfigFile.md) for more information.
+
+## Customization
+
+This parser currently doesn't accept any options.
+
+## Templates
+
+[See here](../templates/colors) for a list of templates and their documentation. If you want to write custom templates, make sure to check the [stencil context documentation](../SwiftGenKit%20Contexts/Colors.md) to see what data is available after parsing.
```

Comment:
Maybe split that in two bullet points.

Also `s/for a list of templates/for a list of bundled templates/` or `for a list of templates bundled with SwiftGen`?